samples/plugins/actian_odbc/dialect.tdd

```diff
@@ -482,7 +482,8 @@
 		<argument type='str' />
     </function>
     <date-function name='DATEDIFF' return-type='int'>
-		<formula>TIMESTAMPDIFF(SECOND,CAST(%2 AS TIMESTAMP(4)),%3)/86400/7</formula>
+		<formula>CAST(ROUND(TIMESTAMPDIFF(DAY,%2,%3)/7.00,0) AS iNTegEr)</formula> <!--Sunil : Added Formula using DAY-->
+		<!--<formula>TIMESTAMPDIFF(SECOND,CAST(%2 AS TIMESTAMP(4)),%3)/86400/7</formula> -->
```

Comment:
don't leave commented-out code

Reply:
will remove the commented-out code in the version of dialect sent for PR